pkg/git/libgit2/checkout_test.go

```diff
@@ -20,13 +20,19 @@ import (
 	"context"
 	"errors"
 	"fmt"
+	"net/url"
 	"os"
 	"path/filepath"
 	"testing"
 	"time"
 
-	git2go "github.com/libgit2/git2go/v31"
+	"github.com/fluxcd/pkg/gittestserver"
+	"github.com/fluxcd/pkg/ssh"
+	git2go "github.com/libgit2/git2go/v33"
 	. "github.com/onsi/gomega"
+	corev1 "k8s.io/api/core/v1"
+
+	"github.com/fluxcd/source-controller/pkg/git"
```

Comment:
```suggestion
	git2go "github.com/libgit2/git2go/v33"
	. "github.com/onsi/gomega"
	corev1 "k8s.io/api/core/v1"
	
	"github.com/fluxcd/pkg/gittestserver"
	"github.com/fluxcd/pkg/ssh"

	"github.com/fluxcd/source-controller/pkg/git"
```

(may apply to other files as well)